bindings/cs/MongoDB.Libmongocrypt.Test/CallbackUtils.cs

```diff
@@ -16,7 +16,7 @@
 
 using System;
 
-namespace MongoDB.Libmongocrypt.Test.Callbacks
+namespace MongoDB.Libmongocrypt.Test
```

Comment:
the CMAKE scripts have problems with folders, I think it can be configured somehow, but by default, the scripts don't see any files except in the solution root. I think having an additional Callback folder it's not the target here, so I removed it

Reply:
I don't understand how this could make any difference.

Isn't CMAKE looking for the DLL and not for individual C# source files?

Reply:
It's not clear for me too. I just see that when I call sh scripts for the project where scripts are placed into `Callbacks` folder. I see this:

>Test Run Successful.
Total tests: 21
     Passed: 21

without Callbacks folder, the number of called tests is:

>Test Run Successful.
Total tests: 17
     Passed: 17

so, the tests in Callbacks folder were just not discovered for some reasons. I would also do not touch it in the scope of the current ticket, but I'm ok to do it later